.github/workflows/metadata_service_deploy_orchestrator_dagger.yml

```diff
@@ -0,0 +1,31 @@
+name: Connector metadata service deploy orchestrator
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - master
+    paths:
+      - "airbyte-ci/connectors/metadata_service/orchestrator/**"
+jobs:
+  connector_metadata_service_orchestrator_deploy:
+    name: Connector metadata service deploy orchestrator
+    runs-on: medium-runner
+    env:
+      CI_GITHUB_ACCESS_TOKEN: ${{ secrets.GH_PAT_MAINTENANCE_OCTAVIA }}
+    steps:
+      - name: Checkout Airbyte
+        uses: actions/checkout@v2
+      - name: Run test for the metadata orchestrator
```

Comment:
If `test` is a `deploy` dependency I think we should declare this dependency directly in the dagger pipeline. This is to follow the philosophy of centralizing the CI logic.